### DISCUSSION TITLE
Error when self-hosting on Vercel

### DISCUSSION THREAD

**Discussion by AustenLamacraft** [giscus/giscus#278] [Q&A] [answered]
I've successfully set up giscus locally but run into the following error when deployed to Vercel the app 403s from GitHub with:
`"error:0909006C:PEM routines:get_name:no start line"`
This seems to suggest a problem with the private key but I'm using the same format that works locally (even setting the variable with the Vercel CLI)
`"-----BEGIN RSA PRIVATE KEY-----\nXXXXXXXXXXXXXXXXXXXXXXXXX\n-----END RSA PRIVATE KEY-----"`
Any idea what's going on? I realise it's my problem but I expect I'm missing something simple...

**Comment by laymonage** [comment, marked as answer]
Have you made sure there's no newlines in the private key?

**Reply by exitcas** [reply]
The newlines have to be replaced with something?

**Reply by laymonage** [reply]
No, just remove them.

**Reply by AustenLamacraft** [reply]
Actually I pasted the version *with* newlines from the GitHub private key `.pem` file and it worked! It's a bit of mystery to me...
